docs/v3-migration-doc.md

```diff
@@ -12,112 +11,62 @@
-  - moved `fluentd.logs.default.sourceName` to `sumologic.logs.defaultFluentd.sourceName`
-  - moved `fluentd.logs.default.sourceCategory` to `sumologic.logs.defaultFluentd.sourceCategory`
-  - moved `fluentd.logs.default.sourceCategoryPrefix` to `sumologic.logs.defaultFluentd.sourceCategoryPrefix`
-  - moved `fluentd.logs.default.sourceCategoryReplaceDash` to `sumologic.logs.defaultFluentd.sourceCategoryReplaceDash`
-  - moved `fluentd.logs.default.excludeFacilityRegex` to `sumologic.logs.defaultFluentd.excludeFacilityRegex`
-  - moved `fluentd.logs.default.excludeHostRegex` to `sumologic.logs.defaultFluentd.excludeHostRegex`
-  - moved `fluentd.logs.default.excludePriorityRegex` to `sumologic.logs.defaultFluentd.excludePriorityRegex`
-  - moved `fluentd.logs.default.excludeUnitRegex` to `sumologic.logs.defaultFluentd.excludeUnitRegex`
-
-- Upgrading Metrics Server to `6.2.4`. In case of changing `metrics-server.*` configuration
-  please see [upgrading section of chart's documentation][metrics-server-upgrade].
-
-- Upgrading Tailing Sidecar Operator helm chart to v0.5.5. There is no breaking change if using annotations only.
-
-- OpenTelemetry Logs Collector will read from end of file now.
-
-  See [OpenTelemetry Collector doesn't read logs from the beginning of files](#opentelemetry-collector-doesnt-read-logs-from-the-beginning-of-files)
-  if you want to keep old behavior.
+## How to upgrade
 
-- Changed `otelagent` from `DaemonSet` to `StatefulSet`
+### Requirements
 
-- Moved parameters from `otelagent.*` to `otelcolInstrumentation.*`
+- `helm3`
+- `kubectl`
+- `jq`
+- `docker`
 
-- Moved parameters from `otelgateway.*` to `tracesGateway.*`
+### Migrating the configuration
 
-- Moved parameters from `otelcol.*` to `tracesSampler.*`
+We've made some breaking changes to our configuration file format, but most of them can be handled automatically by our migration tool.
 
-- Enabled metrics and traces collection from instrumentation by default
-  - changed parameter `sumologic.traces.enabled` default value from `false` to `true`
+You can get your current configuration from the cluster by running:
 
-- Adding `sumologic.metrics.serviceMonitors` to avoid copying values for
-  `kube-prometheus-stack.prometheus.additionalServiceMonitors` configuration
+```bash
+helm get values --output yaml <RELEASE-NAME> > user-values.yaml
+```
 
-- Adding `sumologic.metrics.otelcol.extraProcessors` to make metrics modification easy
+Afterwards, run the upgrade tool:
 
-## How to upgrade
+```bash
+docker run \
+  --rm \
+  -v $(pwd):/values \
+  -i sumologic/kubernetes-tools:2.15.0 \
+  update-collection-v3 -in /values/user-values.yaml -out /values/new-values.yaml
+```
 
-### Requirements
+- At this point you should have `new-values.yaml` in your working directory which
+  can be used for the upgrade:
 
-- `helm3`
-- `kubectl`
-- `jq`
+  ```bash
+  helm upgrade <RELEASE-NAME> sumologic/sumologic --version=3.0.0 -f new_values.yaml
+  ```
```

Comment:
I would add note to look at the migration script output if any manual intervation is needed

Reply:
Honestly, it's probably better not to have this here, as users will run it before they carry out the necessary manual steps from the next section. So this should probably be moved to the end.

Reply:
I moved it to the end and mentioned the output